xml/libvirt_guest_installation.xml

```diff
@@ -132,8 +132,11 @@
       <term><guimenu>Manual install</guimenu></term>
       <listitem>
        <para>
-        To install a specific product version, start typing its name&mdash;for example, <literal>sles</literal>&mdash;and select it once a match appears.
-        Verify that the product's bootable disk or image is available to the virtual machine.
+        This installation method is suitable if you want to create a virtual
+        machine, manually configure its components, and install its OS later.
+        To adjust the VM to a specific product version, start typing its
+        name&mdash;for example, <literal>sles</literal>&mdash;and select it
+        once a match appears.
```

Comment:
```suggestion
        name&mdash;for example, <literal>sles</literal>&mdash;and select the
        desired version once a match appears.
```

Reply:
There were a lot of _it_s in that paragraph; I am trying to reduce ambiguity by removing at least one.